src/yaml/ingestionTrigger/rasterLayerExample.yaml

```diff
@@ -0,0 +1,95 @@
+openapi: "3.0.1"
+info:
+  title: Raster layer examples
+  version: 1.0.0
+components:
+  examples:
+    NewRasterLayerExample:
+      value:
+        metadata:
+          productId: string
+          productName: string
+          productType: Orthophoto
+          productSubType: string
+          description: string
+          srs: string
```

Comment:
should be with value as defined in metadata file

Reply:
done